docs/sections/use-case-requirements.html

```diff
@@ -10,30 +10,230 @@ <h3>Use Cases and Requirements</h3>
+</p>
+
+<p>
+The application will be identified with a <code>CLIENT_ID</code> which MAY be publically shared.
+</p>
+<p>
+The application will be authenticated with a <code>CLIENT_SECRET</code> which MUST NOT be disclosed.
+</p>
+
+<p>
+The organization MAY have more than one DID.
+</p>
+
+<p>
+Each organizatin DID will have DID URLs for <code>assertionMethod</code> and <code>authentication</code> relationships.
+</p>
+
+<p>
+Each DID URL for a given relationship will have a single supported suite type (Ed25519Signature2018, JsonWebSignature2020).
+</p>
+
+<p>
+  As a representative of an organization, I can discover another organization's supported API features from their <code>did:web</code>.
+</p>
+
+</section>
+
+<section>
+  <h5>Discovering Organizational Identifiers</h5>
+
+<p>
+1. Resolve <code>did:web:platform.example:organization:123</code>.
+
+This is trivial, simply do:
+</p>
+
+<pre class="example">
+HTTP GET https://platform.example/organization/123/did.json => didDocument.json
+</pre>
+
+ <p>
+2. Review the <code>serviceEndpoint</code>, <code>assertionMethod</code>, and <code>authentication</code> sections of the DID document.
+ </p>
+<section>
+  <h5>service</h5>
+
+
+<p>
+The serivce block provides a means of communicating or interacting with the DID subject or associated entities via one or more service endpoints.
+</p>
+
+  <p>
+    See <a href="https://www.w3.org/TR/did-core/#dfn-service">service</a>.
+  </p>
+
+<p>
+This entry MUST be present.
+</p>
+<p>
+This entry MUST have an element of type <code>TraceabilityAPI</code> with a <code>serviceEndpoint</code> URL for a VC-API that supports presentation exchange.
+</p>
+
+  <p>For example:</p>
+
+<pre class="example">
+"service": [
+  {
+    "id": "did:web:platform.example:organization:123#traceability-api",
+    "type": "TraceabilityAPI",
+    "serviceEndpoint": "https://platform.example/organization/123"
+  }
+]
+</pre>
+
+<p>This definition implies the following endpoints:</p>
+
+<pre class="example">
+https://platform.example/organization/123/credentials/issue
+https://platform.example/organization/123/credentials/verify
+https://platform.example/organization/123/presentations/prove
+https://platform.example/organization/123/presentations/verify
+https://platform.example/organization/123/presentations/available
+https://platform.example/organization/123/presentations/submissions
+https://platform.example/organization/123/presentations
+</pre>
+
+<p>
+These endpoints require the application to be authenticated:
+</p>
+
+<pre class="example">
+https://platform.example/organization/123/credentials/issue
+https://platform.example/organization/123/presentations/prove
+</pre>
+
+<p>
+These endpoints do not require the application to be authenticated:
+</p>
+
+<pre class="example">
+https://platform.example/organization/123/presentations/available
+https://platform.example/organization/123/presentations/submissions
+</pre>
+
```

Comment:
Moved from the end to here...
```suggestion

<p class="advisement">
Implementations are encouraged to require authentication for all endpoints.
</p>

```

Reply:
This does seem like a better place for this.